### DISCUSSION TITLE
Is it possible to do Geospatial Ops with a Pandas Backend?

### DISCUSSION THREAD

**Discussion by koshy1123** [ibis-project/ibis#4638] [Q&A] [answered]
The [Operation Support Matrix](https://ibis-project.org/docs/3.2.0/backends/support_matrix/) (as of Oct 12, 2022) seems to indicate the `Geo*` operations are supported for the Pandas backend. I tried to use it but hit a `NotImplementedError`:
```
NotImplementedError: Binary operation GeoIntersects not implemented
```

Following the code, I see a list of binary operators currently supported and the Geo* aren't listed. https://github.com/ibis-project/ibis/blob/29d8a2cc6ac835cd9a9a380f7298f188e50a9536/ibis/backends/pandas/execution/constants.py#L61-L79 

Is it fair to assume it was just misspecified in the Operation Support Matrix? If not, does anyone have a simple reproducible working example?

**Comment by gforsyth** [comment, marked as answer]
Hey @koshy1123 -- sorry for the delay in responding here. 
We have a bug in the code that generates the operation support matrix -- The `pandas` and `dask` backends do NOT support geospatial operations via Ibis.  Thanks for bringing this to our attention!

xref #4699
